packages/account/package.json

```diff
@@ -28,7 +28,7 @@
     },
     "dependencies": {
         "@binary-com/binary-document-uploader": "^2.4.7",
-        "@deriv/api-types": "^1.0.11",
+        "@deriv/api-types": "^1.0.85",
```

Comment:
I think the latest version is `1.0.90` or something. but it's not that important.

Reply:
@mohsen-deriv Updated to `1.0.94`, Thanks man 🌹